install-standalone.sh

```diff
@@ -46,10 +46,10 @@
   rm -rf heroku
   rm -rf ~/.local/share/heroku/client
   if [ \$(command -v xz) ]; then
-    URL=https://cli-assets.heroku.com/heroku-\$OS-\$ARCH.tar.xz
+    URL=https://cli-assets.heroku.com/channels/stable/heroku-\$OS-\$ARCH.tar.xz
```

Comment:
I verified this change by running the new script in a docker container and verifying that it installed latest.